spec/RealWorld/Conduit/Users/WebSpec.hs

```diff
@@ -223,6 +223,34 @@ spec =
             Account.email <$>
               (accountFromResponse =<< res) `shouldBe` Right "changed@mail.com"
 
+        it "allows image to be assigned to Nothing" $ do
+          res <-
+            runExceptT $ do
+              account <-
+                ExceptT $ register $ Registrant "secret123" "e@mail.com" "aname"
+              void $ lift $
+                put'
+                  "/api/user"
+                  [(hAuthorization, encodeUtf8 ("Bearer " <> Account.token account))]
```

Comment:
It's probably worth extracting and naming this too :thinking:. That'll likely help even without having a `updateArticle` etc.